src/app/src/components/FilterSidebar.jsx

```diff
@@ -127,7 +130,12 @@ class FilterSidebar extends Component {
     }
 
     render() {
-        const { fetchingFeatureFlags, classes, secondaryColor } = this.props;
+        const { fetchingFeatureFlags, classes, theme } = this.props;
+
+        const actionContrastText = theme.palette.getContrastText(
```

Comment:
The contrast text is needed outside of the styles object because the icons need the color passed in directly, not via a class